qrisk/tests/test_stats.py

```diff
@@ -40,12 +40,71 @@ class TestStats(TestCase):
             '2000-1-1',
             periods=500,
             freq='D'))
-    px_list = np.array(
-        [10, -10, 10]) / 100.  # Ends in drawdown
-    dt = pd.date_range('2000-1-3', periods=3, freq='D')
 
-    px_list_2 = [1.0, 1.2, 1.0, 0.8, 0.7, 0.8, 0.8, 0.8]
-    dt_2 = pd.date_range('2000-1-3', periods=8, freq='D')
+    s_1 = pd.Series(np.array([10, -10, 10]) / 100.,
+                    index=pd.date_range('2000-1-3', periods=3, freq='D'))
+
+    s_2 = pd.Series([1.0, 1.2, 1.0, 0.8, 0.7, 0.8, 0.8, 0.8],
+                    index=pd.date_range('2000-1-3', periods=8, freq='D'))
+
+    s_3 = pd.Series(np.array([1, 10, -2, 2, 3, 2, 1, -10])/100,
+                    index=pd.date_range('2000-1-3', periods=8, freq='D'))
+
+    one = [-0.00171614, 0.01322056, 0.03063862, -0.01422057, -0.00489779,
+           0.01268925, -0.03357711, 0.01797036]
+    two = [0.01846232, 0.00793951, -0.01448395, 0.00422537, -0.00339611,
+           0.03756813, 0.0151531, 0.03549769]
+    df_index_simple = pd.date_range('2000-1-3', periods=8, freq='D')
+    df_index_week = pd.date_range('2000-1-3', periods=8, freq='W')
+    df_index_month = pd.date_range('2000-1-3', periods=8, freq='M')
+
+    d_simple = {'one': pd.Series(one, index=df_index_simple),
+                'two': pd.Series(two, index=df_index_simple)}
+    df_simple = pd.DataFrame(d_simple)
+
+    d_week = {'one': pd.Series(one, index=df_index_week),
+              'two': pd.Series(two, index=df_index_week)}
+    df_week = pd.DataFrame(d_week)
+
+    d_month = {'one': pd.Series(one, index=df_index_month),
+               'two': pd.Series(two, index=df_index_month)}
+    df_month = pd.DataFrame(d_month)
+
+    @parameterized.expand([
+        (simple_rets, 0, 0.33100000000000041),
+        (simple_week_rets, 100, 133.10000000000005),
+        (simple_benchmark, 100, 112.55088100000002)
+        ])
+    def test_cum_returns(self, returns, starting_value, expected):
+        self.assertEqual(
+            stats.cum_returns(
+                returns, starting_value
+            )[-1],
+            expected)
+
+    @parameterized.expand([
+        (simple_rets, stats.WEEKLY, 72),
+        (simple_rets, stats.MONTHLY, 17),
+        (simple_rets, stats.YEARLY, 2),
+        (simple_week_rets, stats.WEEKLY, 499),
+        (simple_week_rets, stats.YEARLY, 10),
+        (simple_month_rets, stats.YEARLY, 42)
+    ])
+    def test_aggregate_returns(self, returns, convert_to, expected):
+        self.assertEqual(
+            len(stats.aggregate_returns(returns, convert_to)),
+            expected)
+
+    @parameterized.expand([
+        (simple_rets, 0.0),
+        (s_3, -0.10000000000000002)
+        ])
```

Comment:
This is really pedantic, but on some tests we do:
```python
@parameterized.expand([
    foo,
    bar
    ])
```

whereas on others, we do
```python
@parameterized.expand([
    foo,
    bar
])
```

can we be consistent on one style? (Not a big deal though)